cmd/clairctl/clair/clair.go

```diff
@@ -0,0 +1,289 @@
+package clair
+
+import (
+	"strconv"
+	"strings"
+  "math"
+	"sort"
+
+	"github.com/coreos/clair/api/v1"
+	"github.com/spf13/viper"
+	"github.com/coreos/clair/cmd/clairctl/xstrings"
+)
+
+var uri string
+var priority string
+var healthPort int
+
+//Report Reporting Config value
+var Report ReportConfig
+
+//VulnerabiliesCounts Total count of vulnerabilities
+type VulnerabiliesCounts struct {
+  Total  int
+  High   int
+  Medium int
+  Low    int
+	Negligible int
+}
+
+//RelativeCount get the percentage of vulnerabilities of a severity
+func (vulnerabilityCount VulnerabiliesCounts) RelativeCount(severity string) float64  {
+  var count int
+  
+  switch severity {
+  case "High":
+    count = vulnerabilityCount.High
+  case "Medium":
+    count = vulnerabilityCount.Medium
+  case "Low":
+    count = vulnerabilityCount.Low
+  }
+  
+  return math.Ceil(float64(count) / float64(vulnerabilityCount.Total) * 100 * 100) / 100
+}
+
+//ImageAnalysis Full image analysis
+type ImageAnalysis struct {
+	Registry  string
+	ImageName string
+	Tag       string
+	Layers    []v1.LayerEnvelope
+}
+
+func (imageAnalysis ImageAnalysis) String() string {
+	return imageAnalysis.Registry + "/" + imageAnalysis.ImageName + ":" + imageAnalysis.Tag
+}
+
+func (imageAnalysis ImageAnalysis) ShortName(l v1.Layer) string {
+	return xstrings.Substr(l.Name, 0, 12)
+}
+
+// CountVulnerabilities counts all image vulnerability
+func (imageAnalysis ImageAnalysis) CountVulnerabilities(l v1.Layer) int {
+	count := 0
+	for _, f := range l.Features {
+		count += len(f.Vulnerabilities)
+	}
+	return count
+}
+
+// CountAllVulnerabilities Total count of vulnerabilities
+func (imageAnalysis ImageAnalysis) CountAllVulnerabilities() VulnerabiliesCounts {
+  var result VulnerabiliesCounts;
+  result.Total = 0
+  result.High = 0
+  result.Medium = 0
+  result.Low = 0
+	result.Negligible = 0
+  
+  for _, l := range imageAnalysis.Layers {
```

Comment:
Yes true. I've found it just before the CoreOSFest and change it to get only the last layer. Should cherry-pick this commit here